jestSetup.js

```diff
@@ -156,7 +172,7 @@ jest.mock('react-native-localize', () => ({
 
 jest.mock('react-native-permissions', () => require('react-native-permissions/mock'))
 
-require('react-native-reanimated/lib/reanimated2/jestUtils').setUpTests()
+jest.mock('react-native-reanimated', () => require('react-native-reanimated/mock'))
```

Comment:
Ok but it would cause this whole PR to have to become a `future!` since it can't compile w/o those changes.

https://github.com/EdgeApp/edge-react-gui/pull/3839